Core/AnalysisInfo.h

```diff
@@ -80,6 +80,38 @@ struct MethodListInfo {
     bool hasDirectSelectors() const;
 };
 
+/**
+ * A description of an Objective-C ivar.
```

Comment:
Pedantic, but: "ivar" being short for "instance variable" should be conveyed at least once somewhere in this code, even though the runtime calls them "ivars". 

In general, I think "instance variable" > "ivar" in comments at least; "ivar" in names/types is fine.

This same comment applies to the next few documentation comments.